mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1115,4 +1145,57 @@
     <string name="search_edit_custom_engine_success_message">Saved %s</string>
     <!-- Text shown when a user successfully deletes a custom search engine -->
     <string name="search_delete_search_engine_success_message">Deleted %s</string>
+
+    <!-- Description text for the Firefox brand logo -->
+    <string name="firefox_logo_description">Firefox logo</string>
+    <!-- Title text shown for the migration screen to the new browser -->
+    <string name="migration_title">Welcome to an all-new Firefox</string>
+    <!-- Description text followed by a list of things migrating (e.g. Bookmarks, History) -->
+    <string name="migration_description">A completely redesigned browser awaits, with improved performance and features to help you do more online.\n\nPlease wait while we update Firefox with your</string>
+    <!-- Text on the disabled button while in progress -->
+    <string name="migration_updating_app_button_text">Updating %s…</string>
+    <!-- Text on the enabled button -->
+    <string name="migration_update_app_button">Start %s</string>
+    <!-- Accessibility description text for completed migration item (e.g. Bookmarks, History) -->
+    <string name="migration_icon_description">Migration status: %s</string>
```

Comment:
Please always explain what placeholder will be replaced with in comments

Reply:
Can someone (possibly @jonalmeida) clarify what this is going to be replaced with in practice? i.e. where is "complete" coming from?

I've never looked at [Kotlin code](https://github.com/mozilla-mobile/fenix/blob/f5715df10476523b8ee1de0ec4d0ff0b2c3887b9/app/src/main/java/org/mozilla/fenix/migration/MigrationProgressActivity.kt#L129-L138), and I have no clue where `migrationText` would get its value from.

Reply:
@Delphine @flodolo the values will be the strings like the example suggests: Bookmarks, History - items that are being migrated from fennec to fenix.

These strings are mapped from the migration type to the string resources [linked here](
https://github.com/mozilla-mobile/fenix/blob/f5715df10476523b8ee1de0ec4d0ff0b2c3887b9/app/src/main/java/org/mozilla/fenix/migration/MigrationProgressActivity.kt#L91-L96).

For example, in the UI there is a check ✔️ icon besides the thing that has completed a migration. This accessibility string was added for that check icon. If you think this unnecessary, we could also make the image as not important for accessibility.

Thanks for pinging me. I was on PTO but I'm happy to help unblock where I can.

Reply:
Thanks for jumping right in. Right now the blocker is the PR in the Fenix repo not being merged yet, resulting in changes to the TOML file ;-)

So, this is an accessibility description of a checkmark displayed after an element (e.g. bookmarks) has been migrated? If that's the case, wouldn't it be more natural to have something like "Migration completed", since the element itself would be already called out before the checkmark? I haven't seen the actual UI, so I might be misunderstanding how it looks like.

Note: Delphine is out for the next 3 months, I'll be taking over the review for this period.

Reply:
> If that's the case, wouldn't it be more natural to have something like "Migration completed", since the element itself would be already called out before the checkmark?

This is a fair point! I'll have this simplified to use your suggestion. :)